pkg/controller/import-controller.go

```diff
@@ -525,6 +525,21 @@ func (r *ImportReconciler) createImporterPod(pvc *corev1.PersistentVolumeClaim)
 		vddkImageName:     vddkImageName,
 		priorityClassName: getPriorityClass(pvc),
 	}
+
+	// Check for old terminating scratch spaces
+	if requiresScratch && podArgs.scratchPvcName != nil {
+		r.log.V(1).Info("Checking for old scratch pvcs")
+		scratchPvc := &corev1.PersistentVolumeClaim{}
+		err := r.client.Get(context.TODO(), types.NamespacedName{Namespace: pvc.GetNamespace(), Name: *podArgs.scratchPvcName}, scratchPvc)
+		if IgnoreNotFound(err) != nil {
+			return err
+		}
+		if scratchPvc.DeletionTimestamp != nil {
```

Comment:
Just making sure if the scratchPvc was found but not deleting its ok? a new one will not be created?

Reply:
I looked it up and it seems ok

Reply:
So with warm import it is currently possible that the scratch space for a previous pod is not gone yet before the next pod is started and if that next pod uses scratch space as well (with this PR it is always the case) then the scratchSpace PVC will never get deleted (in use by new pod) but the pod will never get started (terminating PVC). This code prevents the creation of the new pod until the scratch space PVC is gone.